include/RMGTrackOutputScheme.hh

```diff
@@ -25,6 +25,22 @@
 
 #include "RMGVOutputScheme.hh"
 
+struct RMGTrackEntry {
```

Comment:
you should be able to move this to the `private` section of RMGTrackOutputScheme